src/ansys/dpf/core/common.py

```diff
@@ -228,6 +228,22 @@ class nodal_properties:
     nodal_connectivity = "reverse_connectivity"
 
 
+class face_properties:
+    """Contains strings to define face property fields.
+
+    Attributes
+    ----------
+    element_type = "faces_type"
```

Comment:
property should be `element_type` or `faces_type`?
same for connectivity

Reply:
@rafacanton btw shouldnt it be `face_type` and not `faces_type`
and simply `connectivity`? we are already in the "namespace" `face_properties`

Reply:
@PProfizi For the sake of clarity I've used the same names that the property field have (to not mix things around)